File: php/testcode/benchmark_test_00901.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00901(BenchmarkRequest $req): BenchmarkResponse {
    $conn = ldap_connect('ldap://localhost');
    $base = 'dc=example,dc=com';
    $attr = $req->param('attr');
    $val = $req->param('val');
    $filter = "($attr=$val)";
    $result = ldap_search($conn, $base, $filter);
    $entries = ldap_get_entries($conn, $result);
    return BenchmarkResponse::json(['count' => $entries['count']]);
}

File: rust/testcode/benchmark_test_00901.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let label = req.param("label");

    let html = format!("<svg><text x='10' y='20'>{}</text></svg>", label);

    super::shared::BenchmarkResponse::ok(&html)
}
